tldw_Server_API/app/core/Sharing/unified_share_audit.py

```
@@ -0,0 +1,562 @@
+        resource_id: str,
+        owner_user_id: int,
+        actor_user_id: int | None = None,
+        share_id: int | None = None,
+        token_id: int | None = None,
+        metadata: dict[str, Any] | None = None,
+        ip_address: str | None = None,
+        user_agent: str | None = None,
+        result: str = "success",
+        compatibility_id: int,
+        legacy_share_audit_id: int | None = None,
+        timestamp: Any = None,
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `timestamp`

(ANN401)